packages/contracts-bedrock/test/L1/L1CrossDomainMessenger.t.sol

```diff
@@ -128,6 +128,40 @@ contract L1CrossDomainMessenger_Initialize_Test is L1CrossDomainMessenger_TestIn
         vm.prank(_sender);
         l1CrossDomainMessenger.initialize(systemConfig, optimismPortal2);
     }
+
+    /// @notice Tests that initialize handles zero addresses correctly.
+    function test_initialize_zeroSystemConfig_succeeds() external {
```

Comment:
This is an interesting case. Technically it's a behavior we're OK with, in theory, since we don't really do a lot of validation on inputs, but it's also strange to test that it's allowed. I guess for now we should permit the test to show that this is technically allowed.

Reply:
A more accurate test here would be turning this into a fuzz test, where instead of specifically having zero, it just uses any address and demonstrates that you can input any address and it won't reject that input.